File: Java_Assignment/Assignment_60/programAQ1.java
///////////////////////////////////////////////////////////
//
//  File name : Assignement60Question1.java
//  Description : Write java program to print below pattern.
//  Author : Kartik Ganesh Jare
//  Date : 13/11/25
//  
///////////////////////////////////////////////////////////

/*

 **********
 ****  ****
 ***   ***
 **    **
 *     *
 *     *
 **    **
 ***   ***
 ****  ****
 **********


*/

public class programAQ1
{
    public static void main(String args[])
    {
        int n = 5;

        // Upper part
        for (int i = n; i >= 1; i--)
        {
            // Left stars
            for (int j = 1; j <= i; j++)
            {
            System.out.print("*");
            }

            // Spaces in middle
            int spaces = 2 * (n - i);
            for (int j = 1; j <= spaces; j++)
            {
                System.out.print(" ");
            }

            // Right stars
            for (int j = 1; j <= i; j++)
            {
                System.out.print("*");
            }

            System.out.println();
        }

        // Lower part
        for (int i = 1; i <= n; i++)
        {
            // Left stars
            for (int j = 1; j <= i; j++)
            {
                System.out.print("*");
            }

            // Spaces in middle
            int spaces = 2 * (n - i);
            for (int j = 1; j <= spaces; j++)
            {
                System.out.print(" ");
            }

            // Right stars
            for (int j = 1; j <= i; j++)
            {
                System.out.print("*");
            }

            System.out.println();
        }

    }
}

File: C_Assignment/Assignment_41/programAQ1.c
///////////////////////////////////////////////////////////
//
//  File name : Assignement41Question1.c
//  Description : Write a recursive program which display below pattern..
//  Author : Kartik Ganesh Jare
//  Date : 13/11/25
//  
///////////////////////////////////////////////////////////

/*

Input : 5
Output : * * * * *
Prototype :

*/

#include<stdio.h>

void Display(int iNo)
{
    if(iNo > 0)
    {
        printf("*\t");
        Display(iNo -1);
    }
}

int main()
{
    int iValue = 0;

    printf("Enter number : \n");
    scanf("%d",&iValue);

    Display(iValue);
    
    return 0;
}